pkg/event-manager/controller.go

```diff
@@ -39,6 +39,7 @@ func NewEventController(manager subscriptions.Manager, backend drivers.Backend,
 	c := controller.NewController(controller.Options{
 		ResyncPeriod: config.ResyncPeriod,
 		Workers:      config.WorkerNumber,
+		Namespace:    "events",
```

Comment:
So the namespace attribute is a bit misleading, it was declared in controller struct but not used anywhere, I reused it to indicate which service this controller is for (since in single binary logs are interleaved). Would like me to rename it to something more meaningful? I plan to do some logging improvements in future patches, I want to remove the global logger and make logger instance be passed around, this will allow me to use customized loggers per service (and the need for the hack discussed here will go away)

Reply:
I guess my only suggestion then would be to use a different term, as namespace obviously has other meaning.